apps/site/proxy.ts

```diff
@@ -2,7 +2,7 @@ import createMiddleware from 'next-intl/middleware';
 
 import { availableLocaleCodes, defaultLocale } from '#site/next.locales.mjs';
 
-export default createMiddleware({
+export const proxy = createMiddleware({
```

Comment:
```suggestion
export default createMiddleware({
```

Just leave as it was 😅